bin/show_fe_version.sh

```diff
@@ -22,6 +22,7 @@ curdir=`cd "$curdir"; pwd`
 export STARROCKS_HOME=`cd "$curdir/.."; pwd`
 export PID_DIR=`cd "$curdir"; pwd`
 export LOG_DIR=${STARROCKS_HOME}/log
+source common.sh
```

Comment:
```suggestion
source $STARROCKS_HOME/bin/common.sh
```

use absolute path, otherwise this script can be only run from `bin/` directory.